plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -262,6 +262,29 @@ public void testQueryNullPartitionWithNotPushdownablePredicate()
         assertQuery("SELECT a FROM " + tableName + " WHERE c % 5 = 1", "VALUES (1)");
     }
 
+    @Test
+    public void testPartitionColumnOrderIsDifferentFromTableDefinition()
```

Comment:
Can you pls add a compatibility test with Spark (databricks/delta oss)?

Reply:
Added compatibility tests.